presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -41,6 +41,7 @@
     public static final DataSize DEFAULT_DWRF_STRIPE_CACHE_MAX_SIZE = new DataSize(8, MEGABYTE);
     public static final DwrfStripeCacheMode DEFAULT_DWRF_STRIPE_CACHE_MODE = INDEX_AND_FOOTER;
     public static final int DEFAULT_PRESERVE_DIRECT_ENCODING_STRIPE_COUNT = 0;
+    public static final int DEFAULT_COMPRESSION_LEVEL = Integer.MIN_VALUE;
```

Comment:
This constant should be declare in the presto-hive module. Presto-hive module is the only user and this concept is not specific to ORC.

Reply:
Moved DEFAULT_COMPRESSION_LEVEL to OrcFileWriterConfig located in presto-hive.